nsjail.cc

```diff
@@ -131,10 +202,21 @@ static int listenMode(nsjconf_t* nsjconf) {
 			showProc = false;
 			subproc::displayProc(nsjconf);
 		}
-		int connfd = net::acceptConn(listenfd);
-		if (connfd >= 0) {
-			subproc::runChild(nsjconf, connfd, connfd, connfd);
-			close(connfd);
+		if (pipeTraffic(nsjconf, listenfd)) {
+			int connfd = net::acceptConn(listenfd);
+			if (connfd >= 0) {
+				int in[2];
+				int out[2];
+				if (pipe(in) != 0 || pipe(out) != 0) {
+					PLOG_E("pipe");
+					continue;
+				}
+				nsjconf->pipes.emplace_back(connfd, in[1]);
+				nsjconf->pipes.emplace_back(out[0], connfd);
+				subproc::runChild(nsjconf, in[0], out[1], out[1]);
```

Comment:
Might be good to have an option to filter out one or another - stdout or stderr. Though, it would be used mostly for stderr filtering if ever.

Reply:
This is not going to be used for any filtering :)
Data will be piped in/out by default and this is intended.